test/puppetlabs/trapperkeeper/authorization/rules_test.clj

```diff
@@ -68,6 +52,31 @@
       (testing (str "matching " x)
         (is (= (:rule (rules/match? rule (request "/path/to/resource" x))) rule))))))
 
+(deftest test-matching-query-parameters
```

Comment:
I did that in the integration test, but I'll look at adding one here too I guess